dt-contacts/access-module.php

```diff
@@ -414,7 +414,10 @@ public function dt_details_additional_tiles( $sections, $post_type = "" ){
 
     public function dt_render_field_for_display_template( $post, $field_type, $field_key, $required_tag, $display_field_id ){
         $contact_fields = DT_Posts::get_post_field_settings( "contacts" );
-
+        $disabled = 'disabled';
+        if ( current_user_can( "assign_any_contacts" ) || isset( $post["assigned_to"]["id"] ) && $post["assigned_to"]["id"] == get_current_user_id() ) {
```

Comment:
i think we should use the current_user_can( "update_$post_type") to determine this one

Reply:
Ah yes I changed it in the render function but forgot it here

Reply:
Actually, DT_Posts::can_update( $post_type) is better

Reply:
because there are server caps that could give permission to update